apps/services/user-notification/infra/user-notification.ts

```diff
@@ -1,73 +1,99 @@
+        staging: 'https://identity-server.staging01.devland.is',
+        prod: 'https://innskra.island.is',
+      },
+    })
+    .secrets({
+      FIREBASE_CREDENTIALS: `/k8s/${serviceName}/firestore-credentials`,
+      CONTENTFUL_ACCESS_TOKEN: `/k8s/${serviceName}/CONTENTFUL_ACCESS_TOKEN`,
+    })
+    .liveness('/liveness')
+    .readiness('/readiness')
+    .ingress({
+      primary: {
+        host: {
+          dev: `${serviceName}-xrd`,
+          staging: `${serviceName}-xrd`,
+          prod: `${serviceName}-xrd`,
+        },
+        paths: ['/'],
+        public: false,
+        extraAnnotations: {
+          dev: {
+            'nginx.ingress.kubernetes.io/proxy-buffering': 'on',
+            'nginx.ingress.kubernetes.io/proxy-buffer-size': '8k',
           },
-          paths: ['/'],
-          public: false,
-          extraAnnotations: {
-            dev: {
-              'nginx.ingress.kubernetes.io/proxy-buffering': 'on',
-              'nginx.ingress.kubernetes.io/proxy-buffer-size': '8k',
-            },
-            staging: {
-              'nginx.ingress.kubernetes.io/proxy-buffering': 'on',
-              'nginx.ingress.kubernetes.io/proxy-buffer-size': '8k',
-            },
-            prod: {
-              'nginx.ingress.kubernetes.io/proxy-buffering': 'on',
-              'nginx.ingress.kubernetes.io/proxy-buffer-size': '8k',
-            },
+          staging: {
+            'nginx.ingress.kubernetes.io/proxy-buffering': 'on',
+            'nginx.ingress.kubernetes.io/proxy-buffer-size': '8k',
+          },
+          prod: {
+            'nginx.ingress.kubernetes.io/proxy-buffering': 'on',
+            'nginx.ingress.kubernetes.io/proxy-buffer-size': '8k',
           },
         },
-      })
-      .resources({
-        limits: {
-          cpu: '200m',
-          memory: '384Mi',
-        },
-        requests: {
-          cpu: '15m',
-          memory: '256Mi',
-        },
-      })
-      .grantNamespaces('nginx-ingress-internal')
+      },
+    })
+    .resources({
+      limits: {
+        cpu: '200m',
+        memory: '384Mi',
+      },
+      requests: {
+        cpu: '15m',
+        memory: '256Mi',
+      },
+    })
+    .grantNamespaces('nginx-ingress-internal')
 
 export const userNotificationWorkerSetup = (services: {
-  userProfileApi: ServiceBuilder<'service-portal-api'>
-}): ServiceBuilder<'user-notification-worker'> =>
-  service('user-notification-worker')
-    .image('services-user-notification')
-    .namespace('user-notification')
-    .serviceAccount('user-notification-worker')
+  userProfileApi: ServiceBuilder<typeof serviceWorkerName>
+}): ServiceBuilder<typeof serviceWorkerName> =>
+  service(serviceWorkerName)
+    .image(serviceName)
+    .namespace(serviceName)
+    .serviceAccount(serviceWorkerName)
     .command('node')
     .args('--no-experimental-fetch', 'main.js', '--job=worker')
+    .extraAttributes({
+      dev: { schedule: '30 03 * * *' },
```

Comment:
Does this mean that the worker container will renew at 3:30 or something else?

Reply:
this will trigger a cron job at 3:30 to delete 6 months and older notifications - a cleanup job - but will be implemented later so removing this for now